README.rst

```diff
@@ -94,6 +94,19 @@ This also works with the other parameters like the custom template:
 
     {% flatblock "page.info" evaluated=True using="my_template.html" %}
 
+If you want to simply output the value of the ``content`` field of a flatblock
+without using any template, you can use either options:
+
+.. code-block:: django
+
+    {% flatblock "page.info" evaluated=evaluated using=False %}
```

Comment:
Care to clarify the `evaluated=evaluated` bit?  Perhaps omit it and just mention it's still available?